### DISCUSSION TITLE
Code Runner Like In VSCode

### DISCUSSION THREAD

**Discussion by umair-fa22** [zed-industries/zed#15976] [Feature Requests]
I am looking for a extension that work like
https://github.com/formulahendry/vscode-code-runner.git
this extension of VS Code.
code should run by a shortcut instead of manually compile and run it. or Every language should provide way to run the code as VS Code provides in all Languages mostly.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
